fava/util/date.py

```diff
@@ -94,7 +94,7 @@ def _parse_month(month):
         return months_abbr.index(month)
 
 
-def parse_date(string):
+def parse_date(string, today=None):
```

Comment:
Hmm yes you are right, I should use mock instead. I'll send a PR later today to fix this. Sorry about that.